portal/templates/org_tree.html

```diff
@@ -1,6 +1,6 @@
 {% extends "layout.html" %}
 {% block main %}
-<link rel="stylesheet" href="{{ url_for('static', filename='css/orgTreeView.css') }}">
+<link rel="stylesheet" href="{{ url_for('static', filename='css/orgTreeView.css') }}" type="text/css">
```

Comment:
```
type="text/css"
```
is this still needed? [sounds like](https://stackoverflow.com/questions/5409114/is-type-text-css-necessary-in-a-link-tag) it might not be necessary...